aiortc/codecs/h264.py

```diff
@@ -110,18 +115,147 @@ def parse(self, packet):
 
 
 class H264Encoder:
-    timestamp_increment = 3000
+    timestamp_increment = 6000
```

Comment:
This means 15fps, which is unusual, the defaults I have seen in WebRTC are usually 30fps, no?

Reply:
Humm.. I choose this number because is the average difference between two consecutive packages Chrome sent to me. Based on RFC, the correct is the frame time, but to do it VideoFrame must have a timestamp field